examples/python/keras/callback.py

```diff
@@ -66,6 +67,24 @@ def top_level_task():
 
   model.fit(x_train, y_train, epochs=80, callbacks=[mylr_scheduler, VerifyMetrics(ModelAccuracy.CIFAR10_CNN), EpochVerifyMetrics(ModelAccuracy.CIFAR10_CNN)])
 
+def get_configs():
```

Comment:
This function is duplicated in all tests, maybe we can move it into a common file

Reply:
good idea!